spec/lib/node_rpc_client_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+RSpec.describe NodeRpcClient do
+  let!(:node) { create(:node) }
+
+  before do
+    stub_jrpc_request('options_prober.show.probers', node.rpc_endpoint, { logger: be_present }).and_return([{ id: 1 }])
+  end
+
+  let(:jrpc_response) { NodeRpcClient.new(node.rpc_endpoint).sip_options_probers }
```

Comment:
looks like it's a subject of test

so rewrite it

```ruby
subject do
  NodeRpcClient.new(node.rpc_endpoint).sip_options_probers
end
```

and rename describe to 

```ruby
RSpec.describe NodeRpcClient, '#sip_options_probers' do
```